tests/core/test_version.py

```diff
@@ -18,3 +20,22 @@ def test_get_version(tmpdir):
 def test_no_version_json(tmpdir):
     version = get_version(str(tmpdir))
     assert version is None
+
+def test_env_var_override(tmpdir):
+    content = {"spam": "eggs"}
+    mock.patch.dict(os.environ, { "DOCKERFLOW_VERSION": "foo"})
+    version_json = tmpdir.join("version.json")
```

Comment:
This library uses pytest, so you should use its monkeypatching:

https://docs.pytest.org/en/stable/how-to/monkeypatch.html

```suggestion
def test_env_var_override(tmpdir, monkeypatch):
    content = {"spam": "eggs"}
    monkeypatch.setenv("DOCKERFLOW_VERSION", "foo")
    version_json = tmpdir.join("version.json")
```